.github/workflows/ci-test-limited.yml

```
@@ -256,16 +256,18 @@ jobs:
       - name: Set Commit Message
         env:
           EVENT_COMMITS: ${{ toJson(github.event.commits[0].message) }}
+          COMMIT_INFO_AUTHOR: ${{ github.event.commits[0].author.name }}
         run: |
```

actionlint

shellcheck reported issue in this script: SC2309:warning:1:7: -ne treats this as a variable. Use != to compare as string (or expand explicitly with $var)

(shellcheck)